ts/theme/variables.ts

```diff
@@ -38,6 +42,7 @@ const customVariables = Object.assign(materialVariables, {
   brandDanger: "#CC3333",
   brandLightGray: "#E6E9F2",
   brandDarkGray: "#5C6F82",
+  brandDarkenBlue: "#0073E6",
```

Comment:
derive this from `brandPrimary` using [color](https://www.npmjs.com/package/color#manipulation)